test/WebJobs.Script.Tests.Integration/Host/WebScriptHostManagerTimeoutTests.cs

```diff
@@ -56,31 +56,49 @@ public async Task OnTimeoutException_UsesToken_ManagerKeepsRunning()
             }
         }
 
-        private async Task<TestFunctionHost> RunTimeoutExceptionTest(bool handleCancellation)
+        [Fact]
+        public async Task OnTimeoutException_OOP_HasExpectedLogs()
+        {
+            using (var host = await RunTimeoutExceptionTest(handleCancellation: false, argKey: "inputData", timeoutFunctionName: "TimeoutSync", path: @"TestScripts\Node"))
```

Comment:
Also, add E2E test here: https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_Node.cs ( or any language) that verifies process id is different for worker process and Hostinstance id which is same after the restart.

Reply:
Added validation for node processes, how do I get the HostInstanceId from TestFunctionHost?